apps/public-reference/layouts/application.tsx

```diff
@@ -12,10 +12,19 @@ import {
   UserContext,
 } from "@bloom-housing/ui-components"
 import SVG from "react-inlinesvg"
-import { useContext } from "react"
+import { useContext, useEffect } from "react"
+import { AppSubmissionContext } from "../lib/AppSubmissionContext"
+import { resolversLibrary } from "../lib/resolversLibrary"
 
 const Layout = (props) => {
-  const { profile, signOut } = useContext(UserContext)
+  const userContext = useContext(UserContext)
+  const { profile, signOut } = userContext
+  const applicationContext = useContext(AppSubmissionContext)
+
+  useEffect(() => {
```

Comment:
Are you all using eslint? Looks like issues like this could be caught (and autofixed!) with `react-hooks/exhaustive-deps`: https://www.npmjs.com/package/eslint-plugin-react-hooks#advanced-configuration

Reply:
We are, but unfortunately warnings are not "enforced" in the pre-commit hooks, so if you don't manually run eslint checks, it's easy to miss.